test/relations.jl

```diff
@@ -499,6 +499,16 @@ end
         @test all(saturated.(param_set, ts[RH_sat_mask]))
         @test !any(saturated.(param_set, ts[RH_unsat_mask]))
 
+        # test Clausius Clapeyron relation
+        ts = PhaseEquil_ρeq.(param_set, ρ, e_int, q_tot)
+        L_v = TD.latent_heat_vapor.(param_set, ts)
+        L_s = TD.latent_heat_sublim.(param_set, ts)
+        λ = TD.liquid_fraction.(param_set, ts)
+        L = λ .* L_v .+ (1 .- λ) .* L_s
+        T = TD.air_temperature.(param_set, ts)
+        q_vap_sat = TD.vapor_specific_humidity(param_set, ts)
+        @test all(TD.∂q_vap_sat_∂T(param_set, ts) .≈ L .^ 2 .* q_vap_sat ./ (R_v .* T.^2))
```

Comment:
Update test with correct relation. And this isn't really a test--it just recodes the function. It would be better to test this with AD or small finite differences.